.github/workflows/rust-checks.yml

```diff
@@ -59,6 +59,8 @@ jobs:
       - name: Run tests for examples
         shell: bash
         run: |
+          # todo: use loop xD
```

Comment:
You _could_ reimplement it in the `Makefile` and just call here. See [this](https://github.com/Cardinal-Cryptography/common-amm/blob/main/Makefile#L37-L67) for inspiration.

Reply:
lemme do it in the next PR (#63)